frontend/src/pages/AccountSend.vue

```diff
@@ -116,7 +116,7 @@
             placeholder="vitalik.eth"
             lazy-rules
             :hideBottomSpace="true"
-            :rules="isValidId"
+            :rules="(value: string) => isValidId(value, undefined)"
```

Comment:
(Arbitrary thread)

- Still reviewing, but I found a couple weird scenarios

- It takes a while for the error to show up on this one.
![Screenshot from 2024-01-26 09-43-34](https://github.com/ScopeLift/umbra-protocol/assets/12705146/0270ad41-c57f-406b-80d2-222452a3f408)

- Some state must not be getting cleared
[Screencast from 2024-01-26 11-01-27.webm](https://github.com/ScopeLift/umbra-protocol/assets/12705146/1de2d19d-9f5b-490d-86be-dffc0e712026)

Reply:
Thanks for the review! The warning state not getting cleared issue is resolve here: [f8ff428](https://github.com/ScopeLift/umbra-protocol/pull/640/commits/f8ff428ffc58a3d2219fa6b13724c43538b3e840)

The issue related to the slow error appearance on the batch send form looks like it is also present in the existing release, and appears to be related to the way that the "batchSend"s are "watched" and validations processed.
Might be a good idea to make a separate PR for this issue.

Reply:
Ended up putting the fix for this in the PR after all, as it was so small: https://github.com/ScopeLift/umbra-protocol/pull/640/commits/62a4bf2f9fb0ebf688b32ff9627ab114a334713e

Reply:
The formatting is a little weird in the batch view. This should probably be resolved if possible.

![Screenshot from 2024-01-30 14-48-52](https://github.com/ScopeLift/umbra-protocol/assets/12705146/9c929222-9704-4e84-bfd3-fa368c920576)
![Screenshot from 2024-01-30 14-49-57](https://github.com/ScopeLift/umbra-protocol/assets/12705146/fcb0fa2f-97a3-4b0e-a6e3-5b7fc7cbdd30)
![Screenshot from 2024-01-30 14-50-24](https://github.com/ScopeLift/umbra-protocol/assets/12705146/7fd7e786-8d81-4acb-9475-6a144f2ca68f)

Reply:
Good point.. I think the "error overlapping the ID object below" issue also exist in the current release.. but the warning issue didn't (as that's what this PR was intended to solve.. the missing warning). I should be able to get both resolved in the next commit to this PR.

Reply:
Fix the overlaps on both batch send mobile and desktop..however, if the page is very narrow (just before it transitions to mobile layout) there is still an overlap of the error message onto the warning message .. this is because the base-input component is dropping down a very narrow paragraph (because of the screen width) of the long error text directly beneath it. [8459e97](https://github.com/ScopeLift/umbra-protocol/pull/640/commits/8459e97d09a7a8e2c77bce4b128362e9dfbbfd53)

Reply:
Sorry for the last min nits, but 
1. Could you reduce the distance between the error message and the confusables warning message on Desktop? Ideally to the same distance that they are on the Send page.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/ooGOWJJWJbWqxNVV3Cf5/4c8f2daa-d4be-41b8-8560-404c34164f53.png)

2. Could you make the fonts size of the yellow warning message same as the red error message? Maybe they already are and it's just my browser?

Reply:
Looking good @jferas, I think after these final nits we this will be ready

Reply:
Done: fixed the warning/error layout issue for desktop at various widths:https://github.com/ScopeLift/umbra-protocol/pull/640/commits/f783e6d49a57cb3e9afef52eae28e1e003264a2c
Also fixed a small issue that only showed on an actual device where error overlapped warning: https://github.com/ScopeLift/umbra-protocol/pull/640/commits/964b574f18d446ad024bf310670faf227a8cad8a

Reply:
A couple more nits, 
1. the x-axis padding or margins are a bit larger than the error message.
<img width="441" alt="SCR-20240205-lwoc" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/6cb4ccff-bf57-4bb3-8bda-60b5a10bcb9e">

2. On mobile view, the batch send text box distance between the error and the warning are bit larger than Single Send. 

**Single Send:**
<img width="457" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/5fea5bd7-2d38-4daf-8c17-6b57b86ee4cc">

**Batch Send:**
<img width="402" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/cc6bdec2-bb6f-4340-ab91-fbd7b43d2fde">

3. Weird behavior where if you switch between Single Send and Batch Send while putting in an unregistered address with confusables, Single Page reacts by hiding the error message, and moving the warning message up closer to the address input box. Batch Send page keeps the warning message in the same spot, with a huge margin where the error message would be. 

**Single Send**:
<img width="454" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/8da0f71a-9f0f-401f-939d-47abd57c66c7">

**Batch Send**:
<img width="402" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/75fc9019-2a1d-4e33-ae49-9ea37acb6ac6">

Reply:
Nits 1 and 2 were relatively minor.. done: [8bb038e](https://github.com/ScopeLift/umbra-protocol/pull/640/commits/8bb038e59c6075f371ddc77b9c7c57a53e162b95)

Nit 3 was kind of tricky.. the error message was automatically being removed by Vue on the tab switch because it was part of the recipient ID base-input component, but the warning message (which was generated into its own object was not getting removed on tab switch, so added logic to do that so that error and warning appeared to be handled the same way.. let me know if you think there's a better way of doing this: [462fac7](https://github.com/ScopeLift/umbra-protocol/pull/640/commits/462fac7ad460f99bc3e316367f31e72709192814)

Reply:
I am getting the slightest of overlaps

![Screenshot from 2024-02-12 15-49-05](https://github.com/ScopeLift/umbra-protocol/assets/12705146/2887e84e-423d-4fb3-9a02-fcc86c7d009f)
![Screenshot from 2024-02-12 15-48-53](https://github.com/ScopeLift/umbra-protocol/assets/12705146/09044b74-9270-4a98-bba4-810b702341d3)

Reply:
@jferas will create an issue to track this.